### PR TITLE
Updating the focus color for all elements.

### PR DESCRIPTION
**Pull request by EdwinGuzman**
This is part 1 of 2 for updating the focus style in the app. Partially resolves [SIMPLY-601](https://jira.nypl.org/browse/SIMPLY-601).

The second part would be updating the admin to use these color variables.

### CODE REVIEW

**Comment by aslagle** [comment]
Why not do this in circulation-web? Is this blue meant to be a better focus color no matter what the rest of the color scheme is, or is it specific to NYPL's color scheme?

**Comment by EdwinGuzman** [comment]
I was initially doing this in circulation-web but there were a lot of overrides I didn't want to do in that repo. These colors are specific to NYPL but overall the focus ring color is accessible and should work in any color scheme. It is a bit rough on the header with the dark background, however.

File: packages/opds-web-client/src/stylesheets/_variables.scss [hunk 1]
```diff
@@ -3,14 +3,16 @@ $basefont: $basefontraw * 1px !default;
...
 $darkblue: #234c99;
 $gray: #f5f5f4;
 $light: #ffffff;
 $dark: #403d37;
+$blue-dark: #135772;
 
-// colors
-$focuscolor: orange !default;
+// app colors
+$focuscolor: darken($blue-dark, 5%);
```
[the remark this answers is not in the document]
Comment (EdwinGuzman) [reply packages/opds-web-client/src/stylesheets/_variables.scss]: _In case_ it gets used anywhere else... and I prefer no magic numbers or variables used in operations if I can create a variable for it.

[the remark this answers is not in the document]
Comment (aslagle) [reply packages/opds-web-client/src/stylesheets/_variables.scss]: I meant figure out what the darkened hex value is and make that the variable, so you don't need this darken.

[the remark this answers is not in the document]
Comment (EdwinGuzman) [reply packages/opds-web-client/src/stylesheets/_variables.scss]: That's a good point. I guess we'll never be modifying the percentage.

**Comment by EdwinGuzman** [comment]
Updated the hex color value for `$focuscolor`.

**Comment by EdwinGuzman** [comment]
Re: header color, what do you think about making it white only on the header? It'd only be useful if the background was always dark, however.

screen shot 2018-12-03 at 12 40 54 pm

**Comment by aslagle** [comment]
I guess individual apps could override it for the header if they wanted to.

**Comment by adriana-alter** [comment]
The white looks nice!  (And we're already assuming that the background will always be dark, by virtue of having the text color on the header be white...no?)

**Comment by aslagle** [comment]
We should not be assuming that, it should be based on the theme colors.

**Comment by adriana-alter** [comment]
Changing the header color isn't currently implemented, though...right?  (The header color in the admin interface doesn't actually change if you change the theme colors for a library...or is that not what you meant...?)  I agree that that's definitely something that should be implemented.  But in the meantime, might as well go ahead and make the focus color be white, since the text color already is.

**Comment by aslagle** [comment]
It's not implemented for the admin interface, but it is implemented for the patron web catalog which is also built on this repo.

**Comment by adriana-alter** [comment]
Ah.  That makes sense!

So, in that case: do you think it should be implemented for the admin interface?  (Not in this branch.)

**Comment by aslagle** [comment]
It would be nice to do it for the admin interface but I don't think it's a high priority.

**Comment by adriana-alter** [comment]
Okay--I'll take a look at it at some point.  Aside from anything else, I think it would be nice for admins to be able to preview what they're configuring the patron-facing stuff to look like.

**Comment by EdwinGuzman** [comment]
I agree about a preview but wouldn't know how to implement that in real-time. The library registry and patron web catalog work that Amy did recently helps see changes quite quickly and easily.

For the header, I'll make it white and apps can update it, but should that be a new field in the admin interface or let devs update the focus color themselves? Sounds like this might be getting to be bigger than what it started as.

**Comment by aslagle** [comment]
I don't know if it should be white in this repo - maybe it would be better to make it white in the admin interface only for now.

**Comment by EdwinGuzman** [comment]
Sounds good. I'll leave this as is. Thanks.

**Pull request merged**